lib/src/components/Badge/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { ComponentPropsWithRef, HTMLAttributes } from 'react'
+
+export type BadgeProps = BadgeOptions &
+  ComponentPropsWithRef<'div'> &
+  HTMLAttributes<HTMLDivElement> & {
+    className?: string
```

Comment:
It's include on ComponentPropsWithRef or HTMLAttributes

Reply:
Changed here: [34cd1d1](https://github.com/WTTJ/welcome-ui/pull/2870/commits/34cd1d118d9bb76094de495cc0e78e7cc5e15b28). Thanks @theo-mesnil!